source/blog/2018-02-20-Your-Container-Volumes-Served-By-oVirt.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Your Container Volumes Served By oVirt 
+author: rgolan
+tags: oVirt, oVirt 4.2, open source virtualization,kubernetes, openshift, external-storage, flexdriver, provisioner
+date: Tue Feb 20 08:54:34 IST 2018
+---
+
+> Note: _< 5 minutes read_
+
+When running a virtualization workload on oVirt, a VM disk is 'natively' a disk somewhere on your network-storage.  
+Entering containers world, on k8s or Openshift, there are many options specifically because the workload can be totally stateless, i.e
```

Comment:
Openshift -> OpenShift
(there are more occurrences)

Reply:
@rollandf uploaded new commit please check

Reply:
At least once, you need to mention something like:
Kuberenetes (k8s), then you can use 'k8s' later on.

Reply:
done